int32 pick_slide
int32 pick_grasp
int32 pick_turn
int32 pick_lift
int32 kick_roll
int32 Kick_fire
int32 pass_tee